tests/integrations/test_db_integrations_sql.py

```diff
@@ -289,48 +326,72 @@ def _get_sql_span_for_request(self, weblog_request):
                             return span_child
 
 
-############################################################
-# Postgres: Tracer and Agent validations
-############################################################
-@scenarios.integrations
-class Test_Tracer_Postgres_db_integration(_BaseTracerIntegrationsSqlTestClass):
+class _BaseOtelAgentIntegrationsSqlTestClass(_BaseAgentIntegrationsSqlTestClass):
+    """ Overwrite or add specific methods (on agent interface) for application that has been auto intrumented by Open Telemetry """
+
+    def test_error_msg(self):
+        """ A string representing the error message. """
+        span = self._get_sql_span_for_request(self.requests[self.db_service]["select_error"])
+        assert span["meta"]["error.msg"].strip()
```

Comment:
Easier to read for non python engineers:

```suggestion
        assert len(span["meta"]["error.msg"].strip()) != 0
```

Reply:
Ok. Changed!